simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -418,8 +420,17 @@ public void after() throws IOException {
         Path txt = Paths.get("src/test/resources/" + testName.getMethodName() + ".txt");
         String pngPath = "src/test/resources/" + testName.getMethodName() + ".png";
         String onDisk = Files.exists(txt) ? new String(Files.readAllBytes(txt), StandardCharsets.UTF_8) : "";
+
         boolean txtChanged = !longSummary.equals(onDisk);
 
+        if (System.getenv().containsKey("CI")) { // only strict on CI, locally we just overwrite
+            assertThat(onDisk)
+                    .describedAs("Run tests locally to update checked-in file: %s", txt)
+                    .isEqualTo(longSummary);
```

Comment:
I realized we need this otherwise PRs might forget to update the checked-in txt & png files. Got the failure I wanted here: https://circleci.com/gh/palantir/dialogue/3833